src/botPage/view/blockly/index.js

```diff
@@ -239,12 +237,16 @@ export const load = (blockStr, dropEvent = {}) => {
         const blockType = block.getAttribute('type');
 
         if (!Object.keys(Blockly.Blocks).includes(blockType)) {
-            const error = new TrackJSError(
-                'FileLoad',
-                translate('XML file contains unsupported elements. Please check or modify file.')
+            const error = createError(
+                'InvalidBlockInXML',
+                translate(
+                    `The file you’re trying to open contains unsupported elements in the following block: ${block.getAttribute(
+                        'id'
+                    )}
+                    Please check your file and try again.`
+                )
```

Comment:
it's a wrong pattern for translate content with variables. you can check the correct pattern in `footer.jsx` file

Reply:
you mean that you want to see something like this, without variable?) 

```
globalObserver.emit('Error', createError(
                'InvalidBlockInXML',
                translate(
                    `The file you’re trying to open contains unsupported elements in the following block: ${block.getAttribute(
                        'id'
                    )}
                    Please check your file and try again.`
                )
            ));
```